acceptance/localcluster/localcluster.go

```diff
@@ -426,8 +441,33 @@ func (l *Cluster) Start() {
 	}
 }
 
+// AssertAndStop drains the Events channel and stops the cluster.
+// If the drained events (excluding "start" and "create") don't
+// match the events generated by the operations performed on the
+// nodes in the cluster (restart, kill, ...), the passed Tester
+// receives a fatal error.
+func (l *Cluster) AssertAndStop(t util.Tester) {
+	expected := l.expectedEvents
+	defer l.stop()
+	var events []Event
+	for {
+		select {
```

Comment:
because the channel may be written to. Also (not in this commit yet) we may want to allow for repeated assertions.